src/api/controllers/NotifController.ts

```diff
@@ -0,0 +1,17 @@
+import { Body, CurrentUser, Delete, Get, JsonController, Params, Post } from 'routing-controllers';
+import { ExpoPushMessage, PushTicket } from 'src/types';
+import { NotifService } from '../../services/NotifService';
+
+@JsonController('notif/')
+export class NotifController {
+    private notifService: NotifService;
+
+    constructor(notifService: NotifService) {
+      this.notifService = notifService;
+    }
+
+    @Post()
+    async sendPost( notifRequest : ExpoPushMessage ) {
+        return this.notifService.sendNotifs(notifRequest, {});
+    }
+}
```

Comment:
why are notification changes also included in this PR?

Reply:
sorry i messed up the branches unfortunately, should i go and get rid of them